resources/rds_cluster_parameter_groups.go

```diff
@@ -0,0 +1,207 @@
+				Resolver:    fetchRdsClusterParameterGroupDbParameters,
+				IgnoreError: client.IgnoreAccessDeniedServiceDisabled,
+				Columns: []schema.Column{
+					{
+						Name:        "cluster_parameter_group_cq_id",
+						Description: "Unique CloudQuery ID of aws_rds_cluster_parameter_groups table (FK)",
+						Type:        schema.TypeUUID,
+						Resolver:    schema.ParentIdResolver,
+					},
+					{
+						Name:        "allowed_values",
+						Description: "Specifies the valid range of values for the parameter.",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "apply_method",
+						Description: "Indicates when to apply parameter updates.",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "apply_type",
+						Description: "Specifies the engine specific parameters type.",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "data_type",
+						Description: "Specifies the valid data type for the parameter.",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "description",
+						Description: "Provides a description of the parameter.",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "is_modifiable",
+						Description: "Indicates whether (true) or not (false) the parameter can be modified",
+						Type:        schema.TypeBool,
+					},
+					{
+						Name:        "minimum_engine_version",
+						Description: "The earliest engine version to which the parameter can apply.",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "parameter_name",
+						Description: "Specifies the name of the parameter.",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "parameter_value",
+						Description: "Specifies the value of the parameter.",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "source",
+						Description: "Indicates the source of the parameter value.",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "supported_engine_modes",
+						Description: "The valid DB engine modes.",
+						Type:        schema.TypeStringArray,
+					},
+				},
+			},
+		},
+	}
+}
+
+// ====================================================================================================================
+//                                               Table Resolver Functions
+// ====================================================================================================================
+func fetchRdsClusterParameterGroups(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan interface{}) error {
+	cl := meta.(*client.Client)
+	svc := cl.Services().RDS
+	optsFn := func(o *rds.Options) {
+		o.Region = cl.Region
+	}
+	var input rds.DescribeDBClusterParameterGroupsInput
+	for {
+		output, err := svc.DescribeDBClusterParameterGroups(ctx, &input, optsFn)
```

Comment:
Nit: personal taste, most resources do it this way as well so better consistency
```suggestion
	var input rds.DescribeDBClusterParameterGroupsInput
	for {
		output, err := svc.DescribeDBClusterParameterGroups(ctx, &input,  func(o *rds.Options) {
		o.Region = cl.Region
	})
```

Reply:
Done